src/interfaces/levelEndSceneInterface.ts

```diff
@@ -0,0 +1,35 @@
+import { Monster, Background, AudioPlayer } from "../components";
+import CloseButton from "@components/buttons/close-button";
+import NextButton from "@components/buttons/next-button";
+import RetryButton from "@components/buttons/retry-button";
```

Comment:
Utilize alias and index

Reply:
@bernhardccodev thanks i fixed on my latest commit.